lib/std/os.zig

```diff
@@ -4337,6 +4337,47 @@ pub fn fork() ForkError!pid_t {
     }
 }
 
+/// Returns page size assigned by the Kernel to the process. This size is
+/// constant during process lifetime, but other page sizes might be usable.
+pub fn getDefaultPageSize() error{UnknownPageSize}!usize {
```

Comment:
This function does not belong in std.os (reminder: #5019)

I think it should go into `std.heap.pageSize`.